### PR TITLE
Fixing encryption problems

### PR DESCRIPTION
**Pull request by ronnyek**
I went ahead and did a pretty rudimentary swap of MD5Managed for MD5 .netcore official library. I was able to open files with no security provider, or incorrect password and get expected exceptions, went ahead and added a known PDF to tests and was able to open file with provided password successfully.

### CODE REVIEW

**Comment by ronnyek** [comment]
Admittedly, MD5Managed should be fully gutted, and not have two implementations of MD5 algorithms in place... but this shows things work this way

**Comment by Dids** [comment]
Haven't yet had a chance to properly go over this, but I'm assuming this would fix #3?

File: Didstopia.PDFSharp.Tests/PDFSharpTests.cs [hunk 1]
```diff
@@ -64,31 +66,51 @@ public void TestBasicFunctionality()
...
+
+
+            try
+            {
+                document = PdfReader.Open(PasswordSamplePath, "invalid password", PdfDocumentOpenMode.ReadOnly);
             }
+            
+            catch (Exception ex)
+            {
+                Assert.Equal(ex.Message, "The specified password is invalid.");
```
Comment (Dids) [inline Didstopia.PDFSharp.Tests/PDFSharpTests.cs]: These two parameters should be flipped, so it matches the assertion logic, but also to get rid of the following warning:

```
PDFSharpTests.cs(91,17): warning xUnit2000: The literal or constant value "The specified password is invalid." should be the first argument in the call to 'Assert.Equal(expected, actual)' in method 'TestEncryptionFailsProperly' on type 'PDFSharpTests'. [/home/travis/build/Didstopia/PDFSharp/Didstopia.PDFSharp.Tests/Didstopia.PDFSharp.Tests.csproj]
```

**Review by Dids — CHANGES_REQUESTED** [review]

**Comment by ronnyek** [comment]
Yep, this fixes #3 (at least I think). I can make those minor changes, and I can evaluate whether or not the MD5Managed even needs to be there as its now a full blown library in nuget,

It doesnt open the original pdf with "password" as the password (dont know if thats correct or not, had no way that I could find to just test that it was indeed valid outside of this lib). It doesn't throw that exception, and successfully opens the other PDF I added with valid password "test" as well as fails with expected text. Might make sense to make that a specific exception, might not.

**Comment by ronnyek** [comment]
There we go, still waiting for travis build. Ended up being able to just opt for usage of microsofts fully managed MD5 provider, and all appears to be well.

Updated equality checks, and resolved all resharper warnings other than the name of the test file, which was nitpicky.

**Pull request merged**